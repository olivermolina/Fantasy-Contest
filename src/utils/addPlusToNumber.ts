/**
 * Will add a plus sign to any positive number.
 */
export function addPlusToNumber(num: number) {
  if (num > 0) {
    return `+${num}`;
  }
  return num;
}
