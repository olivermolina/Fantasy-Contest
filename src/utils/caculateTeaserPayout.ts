import { ContestCategory } from '@prisma/client';

/**
 * This function will calculate teaser odds. All teasers currently pay -110 meaning to make 100
 * dollars you have to bet 110.
 *
 * @param stake the amount to bet on the teaser
 */
export function calculateTeaserPayout(
  stake: number,
  contestCategory?: ContestCategory | null,
) {
  const payoutMultiplier = contestCategory?.payoutMultiplier || 0.9;
  return Number((stake * payoutMultiplier).toFixed(2));
}
