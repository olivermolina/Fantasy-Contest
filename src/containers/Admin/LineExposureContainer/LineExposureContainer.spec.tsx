import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { LineExposureContainer } from './LineExposureContainer';
import { trpc } from '~/utils/trpc';

jest.mock('~/utils/trpc', () => ({
  trpc: {
    admin: {
      getLineExposures: {
        useQuery: jest.fn().mockReturnValue({
          isLoading: true,
          data: [],
        }),
      },
    },
  },
}));
describe('LineExposureContainer', () => {
  it('should show a loading screen when the query is loading', () => {
    (trpc.admin.getLineExposures.useQuery as jest.Mock).mockReturnValue({
      isLoading: true,
      data: [],
    });

    //To avoid dayjs dynamic result in snapshots
    jest.useFakeTimers().setSystemTime(new Date('2023-01-01'));

    const { container } = render(<LineExposureContainer />);
    expect(container).toMatchSnapshot();
  });
});
