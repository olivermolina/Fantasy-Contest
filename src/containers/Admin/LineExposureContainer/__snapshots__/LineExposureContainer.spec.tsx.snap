// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineExposureContainer should show a loading screen when the query is loading 1`] = `
<div>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root css-kq809n-MuiBackdrop-root"
    data-testid="backdrop-loading"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  >
    <div
      class="MuiBox-root css-rmz4vk"
    >
      <span
        class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorInherit css-62e83j-MuiCircularProgress-root"
        role="progressbar"
        style="width: 55px; height: 55px;"
      >
        <svg
          class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
          viewBox="22 22 44 44"
        >
          <circle
            class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate MuiCircularProgress-circleDisableShrink css-79nvmn-MuiCircularProgress-circle"
            cx="44"
            cy="44"
            fill="none"
            r="20.2"
            stroke-width="3.6"
          />
        </svg>
      </span>
      <div
        class="MuiBox-root css-1o8dslu"
      >
        <div
          class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-wn51m9-MuiAvatar-root"
        >
          <svg
            height="36"
            preserveAspectRatio="xMidYMid meet"
            role="loading"
            version="1.0"
            viewBox="0 0 176.000000 192.000000"
            width="28"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              fill="#fff"
              id="top"
            />
            <g
              fill="#2463eb"
              stroke="#2463eb"
              transform="translate(0.000000,192.000000) scale(0.100000,-0.100000)"
            >
              <path
                d="M385 1891 c-40 -10 -66 -27 -90 -59 -18 -25 -20 -45 -23 -229 l-3 -203 60 0 61 0 0 173 c0 94 4 178 8 185 7 10 111 12 488 10 l479 -3 3 -187 2 -188 66 0 65 0 -3 204 c-3 224 -5 230 -72 280 -27 20 -39 21 -519 23 -271 1 -505 -2 -522 -6z"
              />
              <path
                d="M153 1309 c-24 -12 -52 -37 -65 -57 l-23 -37 0 -536 0 -535 25 -37 c46 -68 81 -77 289 -77 l182 0 23 53 24 52 -102 5 c-74 4 -109 10 -132 24 -73 44 -86 129 -39 245 l28 71 143 0 143 0 -15 -35 c-22 -52 -19 -56 33 -53 l47 3 28 69 c15 38 28 75 28 83 0 10 -22 13 -97 13 -79 0 -104 4 -130 20 -87 52 -89 127 -12 315 62 150 121 235 193 280 47 29 56 30 185 35 l136 5 16 35 c9 19 20 45 23 58 l7 22 -448 0 c-434 0 -449 -1 -490 -21z"
              />
              <path
                d="M1255 1275 c-14 -30 -25 -57 -25 -59 0 -3 49 -6 109 -8 105 -3 111 -4 140 -32 59 -57 63 -120 15 -241 l-26 -65 -140 0 -140 0 5 23 c3 12 9 32 12 45 7 21 4 22 -37 22 l-45 0 -29 -72 c-16 -40 -30 -76 -32 -80 -2 -5 36 -8 85 -8 119 -1 166 -21 195 -86 26 -60 22 -92 -31 -223 -60 -148 -87 -197 -139 -253 -75 -81 -105 -92 -258 -96 l-131 -4 -16 -46 c-10 -26 -17 -50 -17 -54 0 -5 184 -8 408 -8 403 0 408 0 452 23 24 12 53 35 64 50 21 28 21 41 24 563 3 597 5 578 -66 632 -34 26 -39 27 -194 30 l-158 3 -25 -56z"
              />
              <path
                d="M843 948 c-8 -13 -63 -140 -63 -146 0 -2 23 -2 51 0 l51 3 29 75 c16 41 29 76 29 78 0 9 -90 -1 -97 -10z"
              />
              <path
                d="M933 493 c-48 -110 -48 -103 1 -103 51 0 55 4 89 93 l25 67 -44 0 -45 0 -26 -57z"
              />
            </g>
          </svg>
        </div>
      </div>
    </div>
  </div>
  <div
    class="flex flex-col gap-2 p-2"
  >
    <div
      class="flex flex-row gap-2 items-center"
    >
      <div
        class="MuiFormControl-root MuiTextField-root css-10qkx7n-MuiFormControl-root-MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1sumxir-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink="true"
          for="league"
          id="league-label"
        >
          Select league
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <div
            aria-expanded="false"
            aria-haspopup="listbox"
            aria-labelledby="league-label league"
            class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-jedpe8-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
            id="league"
            role="button"
            tabindex="0"
          >
            NBA
          </div>
          <input
            aria-hidden="true"
            class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
            tabindex="-1"
            value="NBA"
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
            data-testid="ArrowDropDownIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-14lo706"
            >
              <span>
                Select league
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <form>
        <div
          class="css-1m4faer-MuiStack-root"
        >
          <div
            class="MuiFormControl-root MuiFormControl-marginDense MuiFormControl-fullWidth MuiTextField-root css-q2zw81-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1sumxir-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="true"
              for=":r1:"
              id=":r1:-label"
            >
              Start Date
            </label>
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall Mui-readOnly css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                aria-label="Choose date, selected date is Jan 1, 2023"
                aria-readonly="true"
                class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall Mui-readOnly css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
                id=":r1:"
                name="startDate"
                readonly=""
                type="text"
                value="01/01/2023"
              />
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-14lo706"
                >
                  <span>
                    Start Date
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiFormControl-marginDense MuiFormControl-fullWidth MuiTextField-root css-q2zw81-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1sumxir-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="true"
              for=":r3:"
              id=":r3:-label"
            >
              End Date
            </label>
            <div
              class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall Mui-readOnly css-md26zr-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                aria-label="Choose date, selected date is Jan 1, 2023"
                aria-readonly="true"
                class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall Mui-readOnly css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
                id=":r3:"
                name="endDate"
                readonly=""
                type="text"
                value="01/01/2023"
              />
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-14lo706"
                >
                  <span>
                    End Date
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="submit"
          >
            Load
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </form>
    </div>
    <div
      class="MuiBox-root css-8atqhb"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1mxz8qt-MuiPaper-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular bg-blue-500 text-white css-xvd0n5-MuiToolbar-root"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
          >
            Line Exposure
          </h5>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-11xur9t-MuiPaper-root-MuiTableContainer-root"
        >
          <table
            aria-label="Line Exposure"
            class="MuiTable-root css-rqglhn-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="2"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class=""
                    >
                      Odds
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="2"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class=""
                    >
                      Exposure
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                />
              </tr>
              <tr
                class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      ID
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      League
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Name
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Category
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Match Up
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Stat
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Final Stat
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Over
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Under
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Over
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Under
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Game Date & Time
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Game Status
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            >
              <tr
                class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1yhpg23-MuiTableCell-root"
                  colspan="13"
                >
                  <div
                    class="flex flex-col justify-center items-center"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1v0p3qi-MuiSvgIcon-root"
                      data-testid="BrokenImageOutlinedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5v-4.58l.99.99 4-4 4 4 4-3.99L19 12.43V19zm0-9.41-1.01-1.01-4 4.01-4-4-4 4-.99-1V5h14v4.59z"
                      />
                    </svg>
                    No Data
                  </div>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
